### PR TITLE
Updating textbook chapters with latest packages and removing deprecated ignis

### LINKED ISSUE #1470: [3.11 Quantum Teleportation] Chapter uses Ignis (unmaintained since July 2022)
The code example in https://qiskit.org/textbook/ch-algorithms/teleportation.html uses `qiskit.ignis`, which is unmaintained since July 2022 after being deprecated in Dec 2021.

```
from qiskit.ignis.verification import marginal_counts
```

I think `qiskit.ignis.verification.marginal_counts` was replaced by [`qiskit.result.marginal_counts`](https://qiskit.org/documentation/stubs/qiskit.result.marginal_counts.html). If so, this should be an easy fix and a `good first issue`. If not the case, please remove the label.

### LINKED ISSUE #1471: [5.2 Measurement Error Mitigation] Chapter uses Ignis (unmaintained since July 2022)
The code example in https://qiskit.org/textbook/ch-quantum-hardware/measurement-error-mitigation.html uses `qiskit.ignis`, which is unmaintained since July 2022 after being deprecated in Dec 2021.

```python
from qiskit.ignis.mitigation.measurement import complete_meas_cal, CompleteMeasFitter
```

I think `qiskit.ignis.mitigation` was replaced by [qiskit.utils.mitigation](https://qiskit.org/documentation/apidoc/utils_mitigation.html). If so, this should be an easy fix and a `good first issue`. If not the case, please remove the label.

### LINKED ISSUE #1472: [5.4 Measuring Quantum Volume] Chapter uses Ignis (unmaintained since July 2022)
The code example in https://qiskit.org/textbook/ch-quantum-hardware/measuring-quantum-volume.html uses `qiskit.ignis`, which is unmaintained since July 2022 after being deprecated in Dec 2021.

```python
import qiskit.ignis.verification.quantum_volume as qv
```

I think `qiskit.ignis.verification.quantum_volume` was replaced by [qiskit_experiments.library.quantum_volume](https://qiskit.org/documentation/experiments/apidocs/mod_quantum_volume.html). If so, this should be an easy fix and a `good first issue`. If not the case, please remove the label.

### LINKED ISSUE #1474: [5.3 Randomized Benchmarking] Chapter uses Ignis (unmaintained since July 2022)
The code example in https://qiskit.org/textbook/ch-quantum-hardware/randomized-benchmarking.html uses `qiskit.ignis`, which is unmaintained since July 2022 after being deprecated in Dec 2021.

```python
# Import the RB Functions
import qiskit.ignis.verification.randomized_benchmarking as rb
```

I think `qiskit.ignis.verification.randomized_benchmarking` was replaced by [qiskit_experiments.library.randomized_benchmarking	](https://qiskit.org/documentation/experiments/apidocs/mod_randomized_benchmarking.html). If so, this should be an easy fix and a `good first issue`. If not the case, please remove the label.

### LINKED ISSUE #1475: [Lab 3. Accuracy of Quantum Phase Estimation] Lab uses Ignis (unmaintained since July 2022)
The code example in https://qiskit.org/textbook/ch-labs/Lab03_AccuracyQPE.html uses `qiskit.ignis`, which is unmaintained since July 2022 after being deprecated in Dec 2021.

```python
from qiskit.ignis.mitigation.measurement import *
```

I think `qiskit.ignis.mitigation` was replaced by [qiskit.utils.mitigation](https://qiskit.org/documentation/apidoc/utils_mitigation.html). If so, this should be an easy fix and a `good first issue`. If not the case, please remove the label.

### PR DESCRIPTION
**Pull request by divshacker**
# Changes made

I have updated the tutorial with updated packages and removing deprecated ignis from the tutorials. 

# Justification

All the details can be found in issues ranging from #1470 to #1478 
fixes #1470, fixes #1471, fixes #1472, fixes #1474, fixes #1475

### CODE REVIEW

**Comment by divshacker** [comment]
This PR fixes #1470 #1471 #1472 till now. 
Thanks @1ucian0 for reporting these issues.

**Comment by divshacker** [comment]
fixed #1474

**Comment by divshacker** [comment]
fixed #1475

**Comment by 1ucian0** [comment]
I think the `Fixes` notation only works on the first post. Try editing it.

File: content/ch-quantum-hardware/measuring-quantum-volume.ipynb [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "from qiskit.providers.aer.noise.errors.standard_errors import depolarizing_error, thermal_relaxation_error\n",
     "\n",
     "#Import the qv function\n",
-    "import qiskit.ignis.verification.quantum_volume as qv"
+    "import qiskit_experiments.library.quantum_volume as qv"
```
Comment (frankharkins) [inline content/ch-quantum-hardware/measuring-quantum-volume.ipynb:50]:
```suggestion
    "import qiskit.ignis.verification.quantum_volume as qv"
```

Unfortunately this isn't a drop-in replacement as the page relies too heavily on old ignis functionality. I think we leave this as-is; the new textbook will soon have an updated version that uses `qiskit_experiments` (see commit https://github.com/Qiskit/platypus/pull/1505/commits/846a7c5bf7304cf06432577d044a34f7bca9a5bd of PR https://github.com/Qiskit/platypus/pull/1505).

File: content/ch-quantum-hardware/randomized-benchmarking.ipynb [hunk 2]
```diff
@@ -52,7 +52,7 @@
     "from IPython import display\n",
     "\n",
     "# Import the RB Functions\n",
-    "import qiskit.ignis.verification.randomized_benchmarking as rb\n",
+    "import qiskit_experiments.library.randomized_benchmarking as rb\n",
```
Comment (frankharkins) [inline content/ch-quantum-hardware/randomized-benchmarking.ipynb:55]: Same problem here; when running I get 
```
AttributeError: module 'qiskit_experiments.library.randomized_benchmarking' has no attribute 'randomized_benchmarking_seq'
```

If it only takes a couple of lines to get it working then we can do that, for any larger changes we should leave it using ignis and do a re-write in the new textbook.

**Review by frankharkins — CHANGES_REQUESTED** [review]
Thanks! 

I've left a couple of comments about compatibility issues. To avoid disrupting people teaching with this version of the textbook, we're avoiding making large changes. Corrections / small substitutions are fine, but I'm hesitant to make large content changes. Despite deprecation, Ignis does still work and people can still install it with the qiskit version at the bottom of the page to get things working.

**Pull request closed without merging**